len8/models/checker.py

```diff
@@ -93,114 +110,102 @@ def extend(self, extend: bool) -> None:
 
     @property
     def strict(self) -> bool:
-        """If True, raises an error if the check method fails for any
-        reason.
+        """If ``True``, raises an error if the check method fails for
+        any reason.
+
+        Returns:
+            ``bool``
         """
         return self._strict
 
     @strict.setter
     def strict(self, strict: bool) -> None:
         self._strict = strict
 
-    def check(self, *paths: str) -> t.Optional[str]:
-        """Checks to ensure line lengths conform to PEP 8 standards.
-
-        Args:
-            *paths: str
-                The path or paths to check.
-
-        Raises:
-            len8.InvalidPath:
-                If strict mode is set to True and the given path does
-                not exist.
-            len8.BadLines:
-                If strict mode is set to True and the files that were
-                checked contained lines that were too long.
+    def _is_valid(self, path: Path) -> bool:
+        if path.suffix not in (".py", ".pyw"):
+            return False
 
-        Returns:
-            str | None:
-                A formatted string containing the lines that were too
-                long, or None if there were none.
-        """
-        for p in paths:
-            if not os.path.exists(p) and self.strict:
-                raise errors.InvalidPath(p)
+        for e in self.exclude:
+            if path.is_relative_to(e):
+                return False
 
-            if os.path.isfile(p):
-                self._check_file(p)
-            else:
-                self._check_dir(p)
+        return True
```

Comment:
This change does not take into account someone passing an absolute path to the file they want to exclude. 

Assume every file has lines that are too long for minimal reproduction:

Directory structure
```bash
.
├── ignore_me
│   ├── test2.py
│   └── test.py
└── test.py
```

When using a relative path, `./test.py` is ignored as expected.
```bash
$ len8 . -x test.py
2 line(s) are too long:
- /home/jon/repro/ignore_me/test.py, line 1 (119/79)
- /home/jon/repro/ignore_me/test2.py, line 1 (119/79)
```

When using an absolute path for the exclude, `test.py` is not ignored.
```bash
$ len8 . -x $(pwd)/test.py
3 line(s) are too long:
- /home/jon/repro/test.py, line 1 (119/79)
- /home/jon/repro/ignore_me/test.py, line 1 (119/79)
- /home/jon/repro/ignore_me/test2.py, line 1 (119/79)

$ echo $(pwd)/test.py
/home/jon/repro/test.py
```